### PR TITLE
Fix error ObjCBool to Bool under Swift 4.1 Snapshot Linux

### PR DESCRIPTION
**Pull request by damuellen**

### CODE REVIEW

File: Sources/Configs/Config+Directory.swift [hunk 1]
```diff
@@ -47,7 +47,7 @@ extension FileManager {
     fileprivate func isDirectory(path: String) -> Bool {
         var isDirectory: ObjCBool = false
         _ = fileExists(atPath: path, isDirectory: &isDirectory)
-        #if os(Linux)
+        #if os(Linux) && !swift(=>4.1)
```
Comment (Joannis) [inline Sources/Configs/Config+Directory.swift]: I think `#if os(Linux) && swift(<4.1)` would be better

Comment (damuellen) [reply Sources/Configs/Config+Directory.swift]: You're right, it's better to read without negation.

Comment (0xTim) [reply Sources/Configs/Config+Directory.swift]: Unfortunately you aren't going to get that option, the `swift` macro only support `swift(>=X)` 😞

Comment (Joannis) [reply Sources/Configs/Config+Directory.swift]: Really?! I really thought they had normal operators in there

Comment (0xTim) [reply Sources/Configs/Config+Directory.swift]: Nope, only `>=` ['for simplicity'](swift version macro)

**Comment by tanner0101** [comment]
It looks like this has already been fixed. Sorry about missing this one. Thanks!

**Pull request closed without merging**